pkgs/by-name/de/dependency-track/package.nix

```diff
@@ -0,0 +1,110 @@
+{
+  lib,
+  buildNpmPackage,
+  fetchFromGitHub,
+  jre_headless,
+  cyclonedx-cli,
+  makeWrapper,
+  maven,
+  nix-update-script,
+  nixosTests,
+}:
+let
+  version = "4.11.5";
+
+  frontend = buildNpmPackage {
+    pname = "dependency-track-frontend";
+    inherit version;
+
+    src = fetchFromGitHub {
+      owner = "DependencyTrack";
+      repo = "frontend";
+      rev = version;
+      hash = "sha256-YJ5D7hTLlaAgOXz4nT+eh1n5/OulHCsPCN1LAB+1J8M=";
+    };
+
+    npmDepsHash = "sha256-LHK222OzjVHYP4/cVpWfoFSFbfbEt5FNTJry5te3XvQ=";
+    forceGitDeps = true;
+    makeCacheWritable = true;
+
+    # The prepack script runs the build script, which we'd rather do in the build phase.
+    npmPackFlags = [ "--ignore-scripts" ];
+  };
+in
+
+maven.buildMavenPackage rec {
```

Comment:
I believe this `rec` is not needed.

Reply:
It's needed for the `src` reference in line 32.